README.md

```diff
@@ -63,6 +63,16 @@ fn start
 This will start Fn in single server mode, using an embedded database and message queue. You can find all the
 configuration options [here](docs/operating/options.md). If you are on Windows, check [here](docs/operating/windows.md).
 
+### Quickstart with docker-compose
+
+With help of [compose file](./docker-compose.yml) it's easy to bootstrap whole development stack that includes:
```

Comment:
suggestion on language: With Docker Compose it's easy to bootstrap the full Fn development stack that includes the following components: